graylog/Vagrantfile

```diff
@@ -4,46 +4,27 @@
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
-Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "puphpet/debian75-x64"
-  config.vm.box_version = "2.0"
+Vagrant.configure(2)  do |config|
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.hostname = "StepUp-log-Server"
```

Comment:
👍 for changing it to the current project name and removing anachronisms, although I'm not entirely sure about the casing of `StepUp`. 
In repository names and namespaces, `Stepup` is used.

Maybe: `Stepup-log-server` or `Stepup Log Server`?

Shouldn't other references to SuaaS in this repository be changed as well -- for the sake of consistency and ubiquitousness? 
- Vagrantfile for the applications
- `suaas.yml`
- `suaas` directory (incl. reference to it in `.gitignore`)
- inventories (incl. reference to it in `graylog.yml` and `suaas.yml`)
- documentation